scripts/meeting_visit_count.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+import datetime
+import logging
+import sys
+import requests
+from modularodm import Q
+
+from framework.celery_tasks import app as celery_app
+
+from website import settings
+from website.app import init_app
+from framework.transactions.context import TokuTransaction
+from website.models import Tag, Conference
+from website.files.models import StoredFileNode
+from scripts import utils as scripts_utils
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def main(dry_run=True):
+    init_app(routes=False)
+    for conf in Conference.find():
+        count = 0
+        for tag in Tag.find(Q('lower', 'eq', conf.endpoint.lower())):
+            for node in tag.node__tagged.find(Q('is_public', 'eq', True) & Q('is_deleted', 'eq', False)):
```

Comment:
The `tagged` backref has been removed, use `Node.find(<query> & Q('tags', 'eq', tag))`